### PR TITLE
Added all the source files.

### PR DESCRIPTION
**Pull request by Roidujeu**

### CODE REVIEW

File: barbarian.cpp [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+
+void barbarian::on_pushButton_Search_Packages_clicked()
+{
+    QString search_Term = ui->lineEdit_Search->text();
+    QString remotes_Term = ui->lineEdit_Remotes->text();
+    QByteArray search_Term_qbyte = search_Term.toLocal8Bit();
+    QByteArray remotes_Term_qbyte = remotes_Term.toLocal8Bit();
+    char * search_Term_char = search_Term_qbyte.data();
+    char * remotes_Term_char = remotes_Term_qbyte.data();
+    char combined_Buffer[100];
```
Comment (supercmmetry) [inline barbarian.cpp]: Possibility of segmentation fault.

Comment (supercmmetry) [inline barbarian.cpp:47]: Possibility of segmentation fault. Instead use std::string

File: CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+#    if (ANDROID_ABI STREQUAL "armeabi-v7a")
+#        set(ANDROID_EXTRA_LIBS
+#            ${CMAKE_CURRENT_SOURCE_DIR}/path/to/libcrypto.so
+#            ${CMAKE_CURRENT_SOURCE_DIR}/path/to/libssl.so)
+#    endif()
+#endif()
+
+find_package(Qt5 COMPONENTS Widgets REQUIRED)
+
+if(ANDROID)
```
Comment (supercmmetry) [inline CMakeLists.txt:29]: Remove build process for platforms non-specific for the project.

File: main.cpp [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#include "barbarian.h"
+
+#include <QApplication>
+
+int main(int argc, char *argv[])
```
Comment (supercmmetry) [inline main.cpp:5]: Check whether conan is installed in target machine.

File: barbarian.cpp [hunk 4]
```diff
@@ -0,0 +1,262 @@
...
+
+void barbarian::on_pushButton_Remove_clicked()
+{
+    int current_Index_Remove = ui->listWidget_Installed->currentRow();
+    QString current_Remove_Package = ui->listWidget_Installed->item(current_Index_Remove)->text();
+    int Remove_Package_Index = current_Remove_Package.indexOf("/");
+    current_Remove_Package.truncate(Remove_Package_Index);
+    QByteArray current_Remove_Package_QBA = current_Remove_Package.toLocal8Bit();
+    char * curr_Rem_Package_char = current_Remove_Package_QBA.data();
+    char Full_Path_Rem[100];
```
Comment (supercmmetry) [inline barbarian.cpp:256]: Possible segmentation fault. Use std::string instead.

File: barbarian.cpp [hunk 5]
```diff
@@ -0,0 +1,262 @@
...
+            QTextStream Proj_Dir_File_Write(&Proj_Dir_File);
+            Proj_Dir_File_Write << "[requires]" << "\n";
+            Proj_Dir_File_Write << current_Install_Package << "\n\n";
+            Proj_Dir_File_Write << "[generators]" << "\n";
+            Proj_Dir_File_Write << Proj_generator;
+            Proj_Dir_File.close();
+        }
+        QByteArray Proj_Dir_QBA = Proj_Dir.toLocal8Bit();
+        char * Proj_Dir_char = Proj_Dir_QBA.data();
+        char Conan_Install_Command[100];
```
Comment (supercmmetry) [inline barbarian.cpp:204]: Possible segmentation fault. Use std::string instead.

**Review by supercmmetry — CHANGES_REQUESTED** [review]
- Check whether Conan is installed on target machine.
 - Resolve potential segmentation faults.
 - Tidy CmakeList

**Comment by Geek-ubaid** [comment]
@Roidujeu  do the suuggested changes and push a latest commit

**Pull request merged**